test/unit/src/scenes/FogExp2.tests.js

```diff
@@ -5,7 +5,7 @@
 
 import { FogExp2 } from '../../../../src/scenes/FogExp2';
 
-export default QUnit.module( 'FoxExp2', () => {
+export default QUnit.module( 'FogExp2', () => {
```

Comment:
This was a typo that I discovered when doing search&replace. It should be fixed whether or not the PR goes through.

Reply:
Fixed.